pkg/kube/roles.go

```diff
@@ -180,3 +187,79 @@ func userRolesFor(userKind string, userName string, envRoles map[string]*v1.Envi
 	}
 	return answer
 }
+
+// CreateClusterRoleBinding creates or extends a cluster role binding in a given namespace for a service account
+func CreateClusterRoleBinding(kubeClient kubernetes.Interface, namespace string, name string,
```

Comment:
If a clusterrolebinding with the specified `name` already exists, then this method gives no feedback to the caller that it didn't create the cluster. Moreover, the namespace, serviceAccoundName and clusterRoleName passed into the method could differ from the `ClusterRoleBinding` already defined in the cluster, leading to confusion on behalf of the caller when the data in the clusterrolebinding differs from what they have called this method with (because it doesn't create anything).

The return value needs to indicate whether the cluster role was created or not. Either returning a boolean, or maybe even returning the ClusterRoleBinding (that either pre-existed or was created) to the user (or both)